test/Command/ListFixturesCommandTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DotTest\DataFixtures\Command;
+
+use Dot\DataFixtures\Command\ListFixturesCommand;
+use PHPUnit\Framework\MockObject\Exception;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use ReflectionException;
+use ReflectionMethod;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Output\BufferedOutput;
+
+class ListFixturesCommandTest extends TestCase
+{
+    protected ListFixturesCommand|MockObject $listFixturesCommandMock;
+
+    /**
+     * @throws Exception
+     */
+    public function setUp(): void
+    {
+        $this->listFixturesCommandMock = $this->createMock(ListFixturesCommand::class);
+    }
+
+    public function testCreate(): void
+    {
+        $this->assertInstanceOf(ListFixturesCommand::class, $this->listFixturesCommandMock);
+    }
+
+    /**
+     * @throws ReflectionException
+     */
+    public function testCommandWillExecute(): void
```

Comment:
This test is not finished.
Currently it is throwing an error:

> InvalidArgumentException: "test" does not exist

because you need to do the same modifications as with the `ExecuteFixturesCommand` class (make the 3 classes (`Loader`, `ORMPurger` and `ORMExecutor`) injectable, then modify the factory to inject them, then adapt your `ListFixturesCommandTest` tests to mock them where needed).